src/20_Components/amp-fit-text.html

```diff
@@ -0,0 +1,61 @@
+<!--
+  #### Introduction
+  `amp-fit-text` enables you to manage the size and fit of text within a given area
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <!-- #### Setup -->
+  <!--
+    Include the `amp-fit-text` component
+  -->
+  <script async custom-element="amp-fit-text" src="https://cdn.ampproject.org/v0/amp-fit-text-0.1.js"></script>
+  <link rel="canonical" href="<%host%>/components/amp-fit-text/" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <style amp-custom>
+    div.fixedblock { 
+      display: inline-block;
+      height: 200px;
+      width:  200px;
+      background-color: grey;
+    }
+  </style>
+</head>
+<body>
+  <!-- #### Basic Usage -->
+  <!--
+    In each of these examples, the `<amp-fit-text>` element is nested within in a 200x200 grey div block to demonstrate differences in attributes and layout values.
+  -->
+  <!--
+    When `amp-fit-text` is used with a given `height`, `width`, and `layout="responsive"` attribute, the text will scale responsively in the ratio provided but not exceed the size of its parent.
+  -->
+  <div class="fixedblock"><amp-fit-text width="300" height="200" layout="responsive">Lorem ipsum dolor sit amet, has nisl nihil convenire et, vim at aeque inermis reprehendunt.</amp-fit-text></div>
+  <div class="fixedblock"><amp-fit-text width="200" height="200" layout="responsive">Lorem ipsum dolor sit amet, has nisl nihil convenire et, vim at aeque inermis reprehendunt.</amp-fit-text></div>
+  <div class="fixedblock"><amp-fit-text width="100" height="200" layout="responsive">Lorem ipsum dolor sit amet, has nisl nihil convenire et, vim at aeque inermis reprehendunt.</amp-fit-text></div>
```

Comment:
This sample looks broken to me:

<img width="802" alt="screen shot 2016-09-19 at 15 03 03" src="https://cloud.githubusercontent.com/assets/380472/18634724/3c63398c-7e7a-11e6-99d1-8e564226a840.png">